src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,45 @@
+.stopwatch {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  &__center {
+    position: absolute;
+    z-index: 1;
+    height: 30px;
+    width: 30px;
+    background-color: #4b9aaa;
+    border-radius: 50%;
+  }
+
+  &__hand {
+    position: absolute;
+    width: 12px;
+    border-radius: 6px;
+
+    &--minute {
+      height: 30%;
+      background-color: #d15559;
+      bottom: 50%;
+      z-index: -1;
+      transform-origin: bottom;
+      animation-name: move;
+      animation-duration: 3600s;
+      animation-timing-function: linear;
+      animation-iteration-count: infinite;
+    }
```

Comment:
Don't repeat yourself.
It's better to put all styles that are same for `--minute` and `--second` modifiers under `__hand` selector.